.ddev/.setup/scripts/utils.sh

```
@@ -218,22 +289,128 @@ function update_typo3() {
+}
+
+# Function to import SQL data into TYPO3.
+# It checks if the SQL data files exists, and if it does, it imports the SQL
+# data into the TYPO3 database using the MySQL command.
+function import_sql_data() {
+    FIXTURE_DIR="/var/www/html/Tests/Acceptance/Fixtures"
+
+    for DATA_FILE in "$FIXTURE_DIR"/*.sql; do
+        if [ -f "$DATA_FILE" ]; then
+            message yellow "Importing $DATA_FILE..."
+            mysql -h db -u root -p"root" $DATABASE < "$DATA_FILE"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)